src/WrappedGovLst.sol

```diff
@@ -199,11 +198,12 @@ contract WrappedGovLst is ERC20Permit, Ownable {
 
     _burn(msg.sender, _wrappedAmount);
 
-    uint256 _lstAmountUnwrapped = FIXED_LST.convertToRebasing(_wrappedAmount);
+    FIXED_LST.convertToRebasing(_wrappedAmount);
+    uint256 _lstAmountUnwrapped = _calcStakeForShares(_wrappedAmount * SHARE_SCALE_FACTOR);
 
     LST.transfer(msg.sender, _lstAmountUnwrapped);
 
-    emit RebasingUnwrapped(msg.sender, _lstAmountUnwrapped, _wrappedAmount);
+    emit RebasingUnwrapped(msg.sender, _wrappedAmount, _lstAmountUnwrapped);
```

Comment:
@alexkeating The definition of RebasingUnwrapped is
```solidity
  event RebasingUnwrapped(address indexed holder, uint256 lstAmount, uint256 wrappedAmount);
```

Change to this line is wrong, initial version was the correct one.